### ISSUE TITLE
[GUI] Get job details from simulation details.

### ISSUE THREAD

**Issue by berland** [equinor/ert#998]
Suggest to make it possible to view more (all) of the jobs details (`jobs.json` for this particular job) in order to more easily pinpoint the error. `argList` in `jobs.json` is perhaps the most relevant.

![image](https://user-images.githubusercontent.com/306834/91824064-75647e80-ec3a-11ea-9d96-e4efeda1f26e.png)

In this case, the error stems from the input file given to DESIGN_KW, but counting through either the ert-config or jobs.json to figure out which input file is non-trivial (stdout/stderr files do not expose this)

**Comment by jondequinor** [comment]
I totally agree. The `jobs.json` as well as `status.json` information should be accessible (if pertinent) to the user, in both the CLI and the GUI.

**Comment by berland** [comment]
The use case here is to help the user to reproduce the failure by going into the runpath in a terminal and execute a command that should fail.
